dags/common/immersion_facilitee/helpers.py

```diff
@@ -0,0 +1,133 @@
+import datetime
+import logging
+import re
+import time
+from typing import Dict, List
+
+import httpx
+import pandas as pd
+from airflow.models import Variable
+from dateutil.relativedelta import relativedelta
+from sqlalchemy.dialects.postgresql import insert as pg_insert
+from sqlalchemy.orm import Session
+
+from dags.common import db
+from dags.common.anonymize_sensible_data import hash_content, normalize_sensible_data
+from dags.common.immersion_facilitee.models import Conventions
+
+
+logger = logging.getLogger(__name__)
+
+
+def api_client() -> httpx.Client:
+    """
+    Crée un client HTTP pour l'API Immersion Facilitee
+    :return: un client HTTP configuré
+    """
+    return httpx.Client(
+        base_url=Variable.get("API_IMMERSION_FACILITEE_BASE_URL"),
+        headers={"authorization": str(Variable.get("API_IMMERSION_FACILITEE_TOKEN"))},
+        timeout=httpx.Timeout(timeout=5, read=30),
+    )
+
+
+def get_all_items(path: str) -> List[Dict]:
```

Comment:
PI, dans les versions récentes de python tu n'as plus besoin d'importer depuis `typing` et on peux utiliser directement le type de base ;)
```suggestion
def get_all_items(path: str) -> list[dict]
```